kiva/celiagg.py

```diff
@@ -837,14 +839,23 @@ def save(self, filename, file_format=None):
             data = pixels
         img = Image.fromarray(data, 'RGBA')
 
+        ext = (
+            os.path.splitext(filename)[1][1:] if isinstance(filename, str)
+            else ''
+        )
         # Check the output format to see if it can handle an alpha channel.
         no_alpha_formats = ('jpg', 'bmp', 'eps', 'jpeg')
-        if ((isinstance(filename, str) and
-                os.path.splitext(filename)[1][1:] in no_alpha_formats) or
-                (file_format.lower() in no_alpha_formats)):
+        if ext in no_alpha_formats or file_format.lower() in no_alpha_formats:
             img = img.convert('RGB')
 
-        img.save(filename, format=file_format)
+        # Check the output format to see if it can handle DPI
+        dpi_formats = ('jpg', 'png', 'tiff', 'jpeg')
+        if ext in dpi_formats or file_format.lower() in dpi_formats:
+            # Assume 72dpi is 1x
+            dpi = int(self.base_scale * 72)
```

Comment:
Ah I understand 72 now.  This same comment could be added in the previous file rather than wapping it in a variable.  Or both could be wrapped up with a variable